src/components/index.js

```diff
@@ -35,6 +35,7 @@ export { default as FileSelector } from './FileSelector';
 export { default as GMap } from './GMap';
 export { default as GoogleAddressLookup } from './GoogleAddressLookup';
 export { default as HelpText } from './HelpText';
+export { default as HighlightedText } from './HighlightedText';
```

Comment:
we need to export it in the index.d.ts file too in order to add it for typescript users